tools/compare.py

```diff
@@ -244,14 +250,22 @@ def main():
         json2 = gbench.report.filter_benchmark(
             json2_orig, filter_contender, replacement)
 
-    # Diff and output
-    output_lines = gbench.report.generate_difference_report(
-        json1, json2, args.display_aggregates_only,
+    # Optionally, output the diff as JSON
+    diff_report = gbench.report.get_difference_report(
+        json1, json2, args.utest)
+    output_lines = gbench.report.print_difference_report(
+        diff_report,
+        args.display_aggregates_only,
         args.utest, args.utest_alpha, args.color)
     print(description)
     for ln in output_lines:
         print(ln)
 
+    # Optionally, diff and output to JSON
+    if args.dump_to_json is not None:
+        import json
```

Comment:
from https://google.github.io/styleguide/pyguide.html#313-imports-formatting put the imports at the top of the file please.

Reply:
Done. My initial reasoning was to conditionally import modules which are not always necessary.